usr/libexec/greenboot/greenboot-service-monitor

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -eo pipefail
+GREENBOOT_CONFIGURATION_FILE=/etc/greenboot/greenboot.conf
+service_started=true
+if test -f "$GREENBOOT_CONFIGURATION_FILE"; then
+    source $GREENBOOT_CONFIGURATION_FILE
+fi
+
+if [ -n "$GREENBOOT_MONITOR_SERVICES" ]; then
+    services=($GREENBOOT_MONITOR_SERVICES)
+else
+    echo "GREENBOOT_MONITOR_SERVICE not found, using default value"
+    services=( sshd NetworkManager )
+fi
+
+#check all the services listed
+for service in "${services[@]}"; do
+rc="$(systemctl show -p ActiveState --value $service)"
+if [[ $rc != "active" ]]; then
+  echo "<0>${service} did not start"
```

Comment:
please echo the actual ActiveState here. It can be either `active`, `inactive`, `activating`, `deactivating`, or `failed`.